apps/prairielearn/src/sprocs/sync_assessment_modules.sql

```diff
@@ -104,7 +104,7 @@ BEGIN
         DELETE FROM assessment_modules AS am
         WHERE
             am.course_id = syncing_course_id
-            AND am.name NOT IN (SELECT unnest(used_assessment_module_names))
+            AND am.name != ALL(used_assessment_module_names)
```

Comment:
Nit: this isn't really a function so I'd be inclined to include the space between `ALL` and `(`. Ditto elsewhere.

Reply:
Done here and in other references to ANY/ALL in all sprocs in ee5d240f71e26bb0ae921d4460a27160656279eb.